app/src/main/kotlin/com/thundermaps/apilib/android/api/resources/ChannelResource.kt

```diff
@@ -8,10 +8,16 @@ interface ChannelResource {
     suspend fun getChannels(
         parameters: RequestParameters,
         teamId: Long,
-        fields: String = DEFAULT_FIELDS
+        fields: String = DEFAULT_FIELDS,
+        updatedAfter: String
```

Comment:
prefer use less parameter of a function -> To achieve that: we can put teamId, and updatedAfter params to a map in RequestParameters then build the parameters in the request url. 

Example: `index` method of `ReportImpl`